packages/framework/tests/Unit/Model/Product/ProductServiceTest.php

```diff
@@ -239,156 +237,15 @@ public function testDeleteMainVariant()
         $this->assertFalse($variant->isVariant());
     }
 
-    public function testMarkProductForVisibilityRecalculation()
```

Comment:
These tests should be just updated. It is testing of Visibility recalculation, no Price recalculation